src/components/label/examples/label.scss

```diff
@@ -1,4 +1,4 @@
-:host(limel-example-readonly-boolean) {
+:host(limel-example-label) {
```

Comment:
we should also remember to reword the commit title

Reply:
Everything is fixups to your commits and targets your branch :smile: